release_notes/osd_latest_product_updates.adoc

```diff
@@ -0,0 +1,517 @@
+[[release-notes-osd-product-updates]]
+= Latest Product Updates
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+https://www.openshift.com/dedicated/[Red Hat OpenShift Dedicated] provides managed, single-tenant OpenShift environments on the public cloud. Installed and managed by Red Hat, these clusters can provide additional resources as needed, use Red Hat JBoss® Middleware and partner services, integrate with an existing authentication system, and connect to a private datacenter.
+
+Red Hat {product-title} is a Platform as a Service (PaaS) that provides
+developers and IT organizations with a cloud application platform for deploying
+new applications on secure, scalable resources with minimal configuration and
+management overhead. {product-title} supports a wide selection of
+programming languages and frameworks, such as Java, Ruby, and PHP.
+
+See https://www.openshift.com/dedicated[https://www.openshift.com/dedicated] for more information.
+
+[[osd-about-this-release]]
+== About This Update
+
+Red Hat OpenShift Dedicated uses Red Hat OpenShift Container Platform version 3.3 (link:https://access.redhat.com/errata/RHBA-2016:1933[RHBA-2016:1933]) , which is based on
```

Comment:
Could this be a spot to identify the latest cluster version, at least out to the z-stream (e.g., 3.3.0)? Maybe as the first sentence before getting into what it's based on. And only bump this number in subsequent releases when the z-stream changes (e.g., 3.3.1), but not out to the 3.x.x.x release?

Also there's a blank space before the comma here.